quick-integration.hcl

```diff
@@ -0,0 +1,58 @@
+param "converge-bin-dir" {}
```

Comment:
should we put this and the other testing scripts in a `testing` (or something) directory?